.github/workflows/smoke-test.yaml

```
@@ -12,50 +12,63 @@
 jobs:
   docker-build-and-run:
-    runs-on: ['ubuntu-latest']
+    runs-on: [docker-8core-32gb]
 
     env:
       PROJECT_NAME: link
       HEALTH_CHECK_TIMEOUT: 10
       CHECK_INTERVAL: 5
 
-    steps:
+    steps:          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)